smallboard/static/jquery.dataTables.js

```diff
@@ -4486,7 +4486,7 @@
 		var columns = settings.aoColumns;
```

Comment:
Feel free to just delete this file and load from the CDN instead

Reply:
Good idea